rhel/vex/updater_test.go

```diff
@@ -0,0 +1,49 @@
+package vex
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+func TestFingerprintRoundTrip(t *testing.T) {
+	testcases := []struct {
+		name string
+		val  driver.Fingerprint
+		err  bool
+	}{
+		{
+			name: "simple",
+			val:  `one\two\2006-01-02T15:04:05Z`,
+			err:  false,
+		},
+		{
+			name: "date error",
+			val:  `one\two\2006-01-02T15:04:05ZMore`,
+			err:  true,
+		},
+		{
+			name: "etag error",
+			val:  `one\tw\o\2006-01-02T15:04:05ZMore`,
```

Comment:
can you make the time valid to confirm we are failing due to the extra fields and not the time?